app/views/users/pull_requests/show.html.haml

```diff
@@ -0,0 +1,5 @@
+.container
+  %h2= @pull.name
+  %p= link_to @pull.url, @pull.url, target: "_blank"
```

Comment:
__[Brakeman]__

Potentially unsafe model attribute in link_to href


<!-- Comment by [SideCI](https://sideci.com) -->